src/java/io/bazel/rulesscala/scalac/BUILD

```diff
@@ -1,20 +1,32 @@
 java_binary(
-    name = "scalac",
+    name = "scalac_worker",
```

Comment:
do we need to change the target name? The standard is to make the default target the name of the directory.

Reply:
I'll change it back, this change is leftover from when I did it dynamically